packages/beeq/src/components/card/_storybook/bq-card.stories.tsx

```diff
@@ -0,0 +1,99 @@
+          </div>
+          <h4>194</h4>
+        </div>
+        <div class="flex flex-col">
+          <p><span class="font-bold">120</span> days</p>
+          <p class="text-text-secondary">avg resolution</p>
+        </div>
+        <div class="flex flex-col">
+          <p><span class="font-bold">120</span> days</p>
+          <p class="text-text-secondary">avg resolution</p>
+        </div>
+      </div>
+    </bq-card>
+    <bq-card type=${args.type}>
+      <div class="flex flex-col font-medium gap-m">
+          <h6 class="font-bold">Title</h6>
+          <div class="flex flex-col gap-s">
+            <div class="flex flex-row items-center gap-xs">
+              <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+              Lorem Ipsum is simply dummy text 
+            </div>
+            <div class="flex flex-row items-center gap-xs">
+              <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+              Lorem Ipsum is simply dummy text 
+            </div>
+            <div class="flex flex-row items-center gap-xs">
+              <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+              Lorem Ipsum is simply dummy text 
+            </div>
+            <div class="flex flex-row items-center gap-xs">
+              <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+              Lorem Ipsum is simply dummy text 
+            </div>
+          </div>
+        </div>
+    </bq-card>
+    <bq-card type=${args.type}>
+      <div class="flex flex-row gap-m">
+          <bq-icon color="text--brand" size="56" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+          <div>
+            <div class="flex flex-row items-center gap-xs2">
+              <span class="text-text-secondary">Title</span> <bq-icon color="text--brand" size="16" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+            </div>
+            <div class="flex items-end gap-xs">
+              <h4>194</h4>
+              <span class="text-text-success">+24%</span>
+              <span class="text-text-secondary">than last mo.</span>
+            </div>
+          </div>
+      </div>
+    </bq-card>
+    <bq-card type=${args.type}>
+      <div class="flex flex-col items-start gap-m">
+        <bq-icon color="text--brand" size="56" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+        <div class="flex flex-col gap-xs2 max-w-sm">
+          <h6 class="font-bold">Title</h6>
+          <p class="text-text-secondary items-stretch">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.</p>
+        </div>
+        <bq-button appearance="link">Button</bq-button>
+      </div>
+    </bq-card>
+  </div>
+`;
```

Comment:
I would recommend separating each example into one specific Story. You can make the template to accept a content argument, for example:

```ts
import { ifDefined } from 'lit-html/directives/if-defined.js';

const meta: Meta = {
  ...
  // Not part of the component
  content: { control: 'text', table: { disable: true } },
}

const Template = (args: Args) => html`
  <bq-card type=${ifDefined(args.type)}>${ifDefined(args.content)}</bq-card>
`;

export const Default: Story = {
  render: Template,
  args: {
    content: html`
      <div class="flex flex-col gap-6">
        <div>
          <div class="flex flex-row items-center gap-xs2">
            <span class="text-text-secondary">Title</span>
            <bq-icon color="text--brand" size="16" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
          </div>
          <h4>194</h4>
        </div>
        <div class="flex flex-col">
          <p><span class="font-bold">120</span> days</p>
          <p class="text-text-secondary">avg resolution</p>
        </div>
        <div class="flex flex-col">
          <p><span class="font-bold">120</span> days</p>
          <p class="text-text-secondary">avg resolution</p>
        </div>
      </div>
    `;
  }
};
```